src/applications/static-pages/scroll-top-button.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * This button is used for navigating the user back to the top of the page.
+ * It's to be used on long article pages.
+ * There is some complexity in the animation toggling function "scrollListener"
+ * html is located in src/site/components/up_to_top_button.html
+ */
+
+let threeFourthsContainer = '';
+let footer = '';
+const container = document.querySelector('.vsa-top-button-container');
+const distanceOfScrollingBeforeAppearing = 600;
+const BUTTON_CLASSES = {
+  TRANSITION_IN: 'vsa-top-button-transition-in',
+  TRANSITION_OUT: 'vsa-top-button-transition-out',
+  CONTAINER_RELATIVE: 'vsa-top-button-container-relative',
+  TRANSITION_RESET: 'vsa-top-button-transition-reset',
+};
```

Comment:
Since you destructure these properties later on anyway, you could save a few lines by just defining them on their own instead of in `BUTTON_CLASSES` ->

```js
const TRANSITION_IN_CLASSNAME = 'vsa-top-button-transition-in';
// ...
```

That way you won't need to destructure them out of `BUTTON_CLASSES` later on.